agents/labjack/labjack_agent.py

```diff
@@ -71,17 +77,92 @@ def data_to_int32(data):
 def data_to_float32(data):
     return struct.unpack("=f", struct.pack("=I", concatData(data)))[0]
 
+            
+
+class LabJackFunctions:
+    """
+    Labjack helper class to provide unit conversion from analog input voltage
+    """
+    def __init__(self):
+        pass
+    
+    def unit_conversion(self, v, function_info):
+        """
+        Given a voltage and function information from the 
+        labjack_config.yaml file, applies a unit conversion.
+        Returns the converted value and its units.
+        """
+
+        if function_info["user_defined"] == 'False':
+            function = getattr(self, function_info['type'])
+            return function(v)
+        
+        else:     
+            units = function_info['units']
+            value = float(numexpr.evaluate(function_info["function"])) 
+            return value, units
+        
+    def MKS390(self, v):
+        """
+        Conversion function for the MKS390 Micro-Ion ATM
+        Modular Vaccum Gauge. 
+        """
+        value = 1.3332*10**(2*v - 11)
+        units = 'mBar'
+        return value, units
+    
+    def warm_therm(self, v):
+        """
+        Conversion function for SO warm thermometry readout.
+        Voltage is converted to resistance using the LJTick, which
+        has a 2.5V supply and 10kOhm reference resistor. Resistance
+        is converted to degrees Celsius using the calibration curve 
+        for the thermistor, serial number GA10K4D25.        
+        """
+        #LJTick voltage to resistance conversion
+        R = (2.5-v)*10000/v 
+        
+        #Import the Ohms to Celsius cal curve and apply cubic
+        #interpolation to find the temperature
+        reader = csv.reader(open('GA10K4D25_cal_curve.txt'), 
```

Comment:
I'm sure this works, but I'm not sure how, isn't this file within `readout_devices/`?

Reply:
Yep, I think I reorganized after testing, so now I bet it doesn't work. I've fixed it.